TagsterWebAPI/src/Tagster.Infrastructure/EF/Configurations/ProfileConfiguration.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Tagster.Domain.Entities;
+
+namespace Tagster.Infrastructure.EF.Configurations
+{
+    public class ProfileConfiguration : IEntityTypeConfiguration<Profile>
+    {
+        public void Configure(EntityTypeBuilder<Profile> builder)
+        {
+            builder.HasKey(t => t.Id);
```

Comment:
we followed EF Core conventions so those 2 lines are redundant - will work the same without them